activation/handler_test.go

```diff
@@ -1544,6 +1577,145 @@ func TestHandler_HandleParallelGossipAtx(t *testing.T) {
 	require.NoError(t, eg.Wait())
 }
 
+func testHandler_HandleMaliciousAtx(t *testing.T, expectPublish, synced bool) {
```

Comment:
I don't think `expectPublish == synced` makes sense. How about dropping the `expectPublish` boolean and only use `synced`?

```suggestion
func testHandler_HandleMaliciousAtx(t *testing.T, synced bool) {
```

Reply:
It's actually `expectPublish == !synced`, but probably worth changing, yes

Reply:
Fixed